dict.de.css

```
@@ -94,9 +90,13 @@ a.action.load-more::before {
   content: "Alle löschen";
 }
 
+.filter-wrapper .filter:first-child:before {
+  content: "Filters";
```

```suggestion
  content: "Filter";
```
Plural in German doesn't have an `s`